### DISCUSSION TITLE
Issue with Sonoff TH and SI7021 since Tasmota v12.5.0 Peyton (up to Tasmota v13.1.0 Quentin)

### DISCUSSION THREAD

**Discussion by Maker3457** [arendst/Tasmota#19657] [Q&A] [answered]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._
Since Tasmota v12.5.0 Peyton my Sonoff TH with an SI7021 does not measure currectly anymore. Only when rolling back to Tasmota v12.4.0 Peter the measurement is consistent. In my scenario I use the Sonoff TH with the SI7021 to measure the outside temperature my garden. The measured values are sent by MQTT to IOBroker, from there to InfluxDB from where Grafana creates a 24 hours plot. The attached screenshot shows the measurement done with Tasmota v13.1.0 Quentin (until ~19:00) where the plot is unusable. At 19:00 I did a rollback to Tasmota v12.4.0 Peter which shows the correct values over time.  

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): Sonoff TH_____
- [x] Tasmota binary firmware version number used: 12.5.0 - 13.1.0_____
  - [x] Pre-compiled
  - [ ] Self-compiled
- [x] Flashing tools used: OTA _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:

15:59:16.472 CMD: Backlog Template; Module; GPIO 255
15:59:16.498 MQT: freisitz/RESULT = {"NAME":"Generic","GPIO":[32,1,1,1,1,0,0,0,224,320,1,0,0,0],"FLAG":0,"BASE":4}
15:59:16.722 MQT: freisitz/RESULT = {"Module":{"4":"Sonoff TH"}}
15:59:16.976 MQT: freisitz/RESULT = {"GPIO0":{"32":"Button1"},"GPIO1":{"0":"None"},"GPIO2":{"0":"None"},"GPIO3":{"0":"None"},"GPIO4":{"0":"None"},"GPIO5":{"0":"None"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"224":"Relay1"},"GPIO13":{"320":"Led_i1"},"GPIO14":{"1248":"SI7021"},"GPIO15":{"0":"None"},"GPIO16":{"0":"None"},"GPIO17":{"0":"None"}}


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:

16:01:31.337 CMD: Status 0
16:01:31.346 MQT: freisitz/STATUS = {"Status":{"Module":4,"DeviceName":"Freisitz","FriendlyName":["Freisitz"],"Topic":"freisitz","ButtonTopic":"0","Power":0,"PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0,"StatusRetain":0}}
1

```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

![Bildschirmfoto vom 2023-10-02 10-58-35](https://github.com/arendst/Tasmota/assets/120022383/bf2720e9-7b05-44b0-be82-ac1b9697656e)

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Jason2866** [comment, marked as answer]
Solution to find the correct settings for your device is here
https://github.com/arendst/Tasmota/issues/17944#issuecomment-1436657385

**Reply by Maker3457** [reply]
Ok, thanks for the hint - will follow up there. Did already some rather positive tests with that DhtDelay parameter.

**Comment by Maker3457** [comment]
Since I set "DhtDelay 480,40" the temperature measurement works as expected.
